snap/README.md

```diff
@@ -0,0 +1,5 @@
+# Snap
+
+## InfluxDB - Grafana
+All files for running Snap with InfluxDB and Grafana are from www.github.com/intelsdi-x/snap/examples/influxdb-grafana.
```

Comment:
www.github.com/intelsdi-x/snap/examples/influxdb-grafana isn't a valid URL. Should it be https://github.com/intelsdi-x/snap/tree/master/examples ?